extensions/reviewed/Bounce.json

```
@@ -469,6 +492,47 @@
             }
           ],
           "objectGroups": []
+        },
+        {
+          "description": "the number of times this object has bounced off another object of the object.",
+          "fullName": "Number of times this object has bounced off another object",
+          "functionType": "ExpressionAndCondition",
+          "group": "Bounce configuration",
```

It's a state not a configuration.
```suggestion
          "group": "",
```